packages/neuron-wallet/src/keys/tool.ts

```diff
@@ -2,25 +2,27 @@ import { BIP32 } from 'bip32'
 import { Child } from './keystore'
 
 // BIP44: https://github.com/satoshilabs/slips/blob/master/slip-0044.md
-const PURPOSE = "44'"
-// 360 is tentative value
-const COIN_TYPE_TESTNET: string = "360'"
-const ACCOUNT = "0'"
-// external address
-const CHANGE = 0
+enum Constant {
+  PURPOSE = "44'",
+  // 360 is tentative value
+  COIN_TYPE_TESTNET = "360'",
```

Comment:
enum should be named in `PascalCase`

Reply:
OK

Reply:
Isn't 'Constant' too generic a name for enum?

Reply:
Indeed! I will refactor later.